src/funnels/Swapper.sol

```diff
@@ -38,14 +38,15 @@ contract Swapper {
     address   public immutable buffer; // Contract from which the GEM to sell is pulled and to which the bought GEM is pushed
 
     struct PairLimit {
+        uint128 cap; // Maximum amount of src token that can be swapped each hop for a src->dst pair
         uint64  hop; // Cooldown one has to wait between each src to dst swap
+        uint128 amt; // Amount already swapped during the last hop
```

Comment:
The comment is wrong as it is the amount remaining.
Maybe rename to something like `left` everywhere? (to remove also remove confusion with the StableDepositor's amts, which are something different).

Reply:
Fixed.

Reply:
And changed variable to `due`.